packages/terra-base/src/Base.scss

```diff
@@ -31,3 +31,39 @@ body {
 *::after {
   box-sizing: inherit;
 }
+
+a {
+  background-color: transparent; /* Remove the gray background on active links in IE 10. */
+  color: var(--terra-link-color, #0065a3);
+  outline: none;
+  text-decoration: underline;
+}
+
+// stylelint-disable selector-no-qualifying-type
+a:visited,
+a.is-visited {
+  color: var(--terra-link-visted-color, #0065a3);
+}
+
+a:hover,
+a.is-hovered {
+  color: var(--terra-link-hover-color, #004c76);
+}
+
+a:focus,
+a.is-focused {
+  background-color: var(--terra-link-focus-background-color, rgba(136, 168, 239, 0.5));
+  border-radius: 0.2em;
+  color: var(--terra-link-focus-color, #004c76);
+}
+
+a:active,
+a.is-active {
+  color: var(--terra-link-active-color, #0065a3);
+}
+
+a[aria-disabled='true'] {
```

Comment:
We should stop disabled hyperlinks from triggering focus styling.

Reply:
Actually, after glancing at the UX callouts, seems that focus should be allowed in all states. Sorry, ignore the previous comment. If we truly want a disabled link in the future, we could remove the href altogether, or have a prop to change the tab-index to a negative value. Seems that having a disabled hyperlink doesn't make sense in the first place though ([source](https://css-tricks.com/how-to-disable-links/)).

Reply:
@yuderekyu After discussing with @bjankord and @neilpfeiffer we decided to move forward with just the styles we have for disabled link as now and a new issue will be logged for a fully functional disabled link.